react-native/react/actions/update.js

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+
+import * as Constants from '../constants/update'
+import engine from '../engine'
+import moment from 'moment'
+
+import {updateUi} from '../constants/types/keybase_v1'
+import type {Update} from '../constants/types/flow-types'
+import type {ShowUpdateAction, RegisterUpdateListenerAction, OnCancelAction, OnSkipAction, OnSnoozeAction, OnUpdateAction} from '../constants/update'
+import type {Dispatch} from '../constants/types/flux'
+
+let updateResponse: ?(result: any/* UpdatePromptRes */) => void
```

Comment:
If it's not an error to not pass all the values, then those values should be marked maybes in the avdl. Any reason why they aren't?

I'm okay making a separate ticket for that and doing that in another PR.